### ISSUE TITLE
pip issues 

### ISSUE THREAD

**Issue by twpDone** [Neo23x0/yarGen#8]
[Linux][Kali]
# TLDR : FIX

If you got a 
`AttributeError: 'module' object has no attribute 'PY2'`

**Just try to update PIP**
`wget https://bootstrap.pypa.io/get-pip.py`
`python get-pip.py`
`rm get-pip.py`
`sudo pip install scandir lxml naiveBayesClassifier pefile`
# Error

pip raise an Exception for scandir lxml naiveBayesClassifier
`pip install scandir` 
Downloading/unpacking scandir
  Downloading scandir-1.2.tar.gz
Cleaning up...
Exception:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 122, in main
[...]
  File "/usr/lib/python2.7/dist-packages/pip/req.py", line 285, in setup_py
    if six.PY2 and isinstance(setup_py, six.text_type):
AttributeError: 'module' object has no attribute 'PY2'
`
# Unworking fixes

Note that, in order to avoid errors, you may want to use :
`sudo pip install pefile`
Then run
`sudo pip3 install scandir lxml naiveBayesClassifier`

**_But this raise errors if you run yaraGen _**

**with python2**
`ImportError: No module named scandir`

**with python3**
`except Exception, e:`
`SyntaxError: invalid syntax`

I searched for conflict between pip and pip3, runned apt-get update && apt-get upgrade but the issue was still there. 
So i checked the official pip website and checked my pip version, which wasn't up to date.
The Upgrade made the trick.

**Comment by twpDone** [comment]
Fix doesn't work properly, i'll get back at you when i found a working fix

**Comment by twpDone** [comment]
Fix don't work, i'll get back at you when a solution is found

**Comment by twpDone** [comment]
Kali hadn't the last version of pip installed
Edited first comment
